util/lua/lua.go

```diff
@@ -239,7 +239,7 @@ func (vm VM) ExecuteResourceActionDiscovery(obj *unstructured.Unstructured, scri
 func isActionAvailable(actionsMap interface{}) bool {
 	actions, ok := actionsMap.(map[string]interface{})
 	if !ok {
-		return false
+		return true
```

Comment:
Currently, this change would mean that when a `discovery.lua` script returns a malformed or unexpected result, then the action would be displayed as available.

Reply:
We don't have many actions yet, only deployment restart and rollout resume. But in both cases available/not available is UI only feature. The actions scrips are idempotent and work correctly in both available/not available states.